README.md

```diff
@@ -1 +1,11 @@
-# angular-compile-link-lab
\ No newline at end of file
+# Compile and link
+
+## Objectives
+
+- Explore the APIs and instances given to us through compile and link
+
+## Instructions
+
+Take the directive in this repo, and add a link function to it.
+
+Experiment with `compile` and `link` to the change the text in our directive to anything else.
```

Comment:
Make these instructions more explicit - tell the student exactly what the text should be changed to. 

Are there two different tasks that they could do here - one that would use `link` and one that would use `compile`?